hollow-jar/full/src/main/java/org/wildfly/swarm/ts/hollow/jar/full/RemoteEJBServlet.java

```diff
@@ -0,0 +1,23 @@
+package org.wildfly.swarm.ts.hollow.jar.full;
+
+import java.io.IOException;
+
+import javax.ejb.EJB;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.wildfly.swarm.ts.hollow.jar.full.remoteejb.RemoteEJB;
+
+@WebServlet("/remoteejb")
+public class RemoteEJBServlet  extends HttpServlet {
+
+    @EJB (lookup="java:module/RemoteEJBImpl")
```

Comment:
Are we sure this really gets a remote proxy?

Also extra space between `@EJB` and opening `(`.

Reply:
Maybe should invoke the EJB remotely from the test code.